persistence/indexer/doc/README.md

```diff
@@ -1,16 +1,16 @@
 # Transaction Indexer
 
-`txIndexer` implementation uses a `KVStore` (interface) to index the transactions.
+`txIndexer` implementation uses a `KVStore` (interface) to txIndex the transactions.
```

Comment:
Is this change intentional? There are no changes in the indexer business logic

Reply:
Reverting!